libraries/Microsoft.Bot.Configuration/BotConfiguration.cs

```diff
@@ -110,20 +121,41 @@ public static async Task<BotConfiguration> LoadAsync(string file, string secret
         }
 
         /// <summary>
-        /// Save the file with secret
+        /// Load the configuration from a .bot file.
         /// </summary>
-        /// <param name="secret">secret for encryption</param>
+        /// <param name="file">Path to bot file. </param>
+        /// <param name="secret">Secret to use to decrypt the file on disk. </param>
+        /// <returns>BotConfiguration. </returns>
```

Comment:
This should be a link to the BotConfiguration Type so the docs generate hyperlinks.